### ISSUE TITLE
[FEATURE] Script "enemy has no essence"

### ISSUE THREAD

**Issue by Baan12** [Shilo/SD2-Suggestions#13]
**Is your feature request related to a problem? Please describe.**
earlier i was helping somebody to set up scripts and we discrovered an issue:
this person was clearing rooms >2 with dark lord's aoe but wanted to ransack in rooms with less enemies.
he then decided to default hit with dark lord in levels with 2 enemies but half of the time he would crit and one shot the essence mob. we werent able to find a way to make him hit the monster that had no essence instead

**Describe the solution you'd like**
would be nice to be able to script "enemy has no essence"

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by Shilo** [comment]
I agree on this. You can probably do tricky workarounds by doing triggers after "enemy has essence" to simulate "no essence", but it gets massively more difficult in that case. So having "enemy has no essence" would definitely be useful.

**Comment by firered33** [comment]
This is already coming next update. Details here: https://trello.com/c/rl7MmClU
